examples/docking/docking.toml

```diff
@@ -11,6 +11,7 @@ order = ["topoaa",
 
 # directory in which the scoring will be done
 run_dir = "run1"
+ncores = 8
```

Comment:
If we have this parameter here does it mean it always needs to be passed in the configuration file? This won't make sense once we reach the actual execution methods of HPC/GRID

Reply:
No, it doesn't mean that. It means that you can add it to control the number of CPUs. But if you don't add it, HADDOCK3 will use all CPUs available minus 1.

Note, it is also possible to have the `ncores` specified for each module separately. If that is not specified, the program will use the global value.